CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # bedrock-vue-wallet ChangeLog
 
+## 22.2.0 - 2023-11-dd
+
+### Fixed
+- Allow credentials with any description length to align title, description, and
+  image correctly in credential list.
```

Comment:
This is a breaking change because anyone who had used the components in the current version may have been relying on the max-width to apply without setting it in the parent component.

```suggestion
## 23.0.0 - 2023-11-dd

### Changed
- **BREAKING**: Allow credentials with any description length to align title,
  description, and image correctly in credential list.
```

Reply:
Have we tested how things change in other places where this component is in use?

Reply:
I forgot to update the `tested` portion of the description above, that's been fixed. 

I have test both requesting a verifiable presentation as well as issuing a credential from the vc playground to a local veres wallet.

> Screenshot from issuing credential locally
>
> <img alt="screenshot" width="400" src="https://github.com/digitalbazaar/bedrock-vue-wallet/assets/56396286/009cbfb1-2976-450d-8da9-42f7e63fb260" />

Reply:
@jameseaster,

But what about other views? Like the credentials page and so on? Have we checked all the places where this component was used previously?

Reply:
@dlongley Great questions.

It looks to me that this change affects the `CredentialSelect` component and the `CredentialCompactBundle` component which are both rendered in `CredentialsList` component (see line 3 and line 9 in CredentialsList.vue) if the `compact` prop is true (see line 2 in CredentialsList.vue).

If we look for the use of `CredentialsList` with the `compact` prop set to true value we'll see it mainly used in these two places: 
- StoreCredentials.vue
- ShareCredentials.vue

It is also used in `ShareReview.vue` line 26, however the `ShareReview` component is only used inside of `ShareCredentials.vue` line 35 which is already listed above.

<br/>

<br/>

Here are the images of the two places that this is used in Veres Wallet:

> StoreCredentials.vue line 35 rendered in ChapiExchangePage line 33 (../credential-handler/exchange)
>
> <img alt="store" width="600" src="https://github.com/digitalbazaar/bedrock-vue-wallet/assets/56396286/b77abd1c-deb1-4c1c-95b8-b9c4dd611b6a" />

> ShareCredentials.vue line 35 rendered in ChapiExchangePage line 27 (../credential-handler/exchange)
>
> <img alt="share" width="600" src="https://github.com/digitalbazaar/bedrock-vue-wallet/assets/56396286/5e33dab1-3c93-4ef9-9522-47f9c3ba8fc1" />